lib/config.js

```diff
@@ -1,9 +1,14 @@
+var service_host = process.env.MONGO_SERVICE_HOST
+
 var database = {
-    url: 'mongodb://localhost:27017/pacman',
+    url: `mongodb://${service_host}:27017/pacman`,
```

Comment:
MONGO_SERVICE_HOST is injected into the pod automatically as part of the k8s downward api